### PR TITLE
News about assets in 2.3

### PR DESCRIPTION
**Pull request by morganen**
- Add a missing paragraph about asset collection attribute
- Add a new paragraph about asset upload in the product form
- Add a new paragraph about asset ordering in the product form
- Add a new paragraph about asset preview in the product form

### CODE REVIEW

File: content/md/products-super-power/work-on-a-product.md [hunk 1]
```diff
@@ -95,6 +95,76 @@ From the product form:
 
 If you do not see `+` icon, you might not have the appropriate rights to add an option, please refer to [Manage your user roles](/articles/build-your-user-roles.html).
 
+## Work with assets in an asset collection attribute
```
Comment (KmilleF) [inline content/md/products-super-power/work-on-a-product.md]: Add the "Enterprise Edition" mention

File: content/md/products-super-power/work-on-a-product.md [hunk 2]
```diff
@@ -95,6 +95,76 @@ From the product form:
...
+
+In the product sheets, you can have some asset collection attributes which look like this:
```
Comment (KmilleF) [inline content/md/products-super-power/work-on-a-product.md]: we use product form in the helpcenter

Comment (KmilleF) [reply content/md/products-super-power/work-on-a-product.md]: Could be nice to add a link to the asset article

File: content/md/products-super-power/work-on-a-product.md [hunk 3]
```diff
@@ -95,6 +95,76 @@ From the product form:
...
+You can link assets to your product using this attribute. To do that, follow these steps:
+
+1. Go to your product form
+1. Scroll down towards the asset collection attribute
+1. Click on `Manage assets` to access to the asset picker and the basket
+1. Pick the assets you want to link to your product, and save.
+
+![basket](../img/basket.png)
+
+### Upload assets directly from the asset collection
```
Comment (KmilleF) [inline content/md/products-super-power/work-on-a-product.md]: I assume "From the product form" would be clearer for end users.

Comment (KmilleF) [reply content/md/products-super-power/work-on-a-product.md]: "to access the asset picker"

File: content/md/products-super-power/work-on-a-product.md [hunk 4]
```diff
@@ -95,6 +95,76 @@ From the product form:
...
+
+From the 2.3 version, you can upload assets directly from the Asset collection in the Product Form. You don't need to quit the product form anymore.
+
+1. Click on the new button `Upload assets` in the asset collection, then you will retrieve the mass upload screen.
```
Comment (KmilleF) [inline content/md/products-super-power/work-on-a-product.md]: "A mass upload screen will be prompted"?

File: content/md/products-super-power/work-on-a-product.md [hunk 5]
```diff
@@ -95,6 +95,76 @@ From the product form:
...
+1. Drag and drop your files or click on the image to chose your assets on your browser.
+1. Click on `Upload`.
+1. Then click on `Import`.
+1. You may will have to wait a bit while the asset import. Refresh the page when you receive the notification, and you will see assets linked to your product in the asset collection.
```
Comment (KmilleF) [inline content/md/products-super-power/work-on-a-product.md]: "you may have"?

File: content/md/products-super-power/work-on-a-product.md [hunk 6]
```diff
@@ -95,6 +95,76 @@ From the product form:
...
+:::
+
+### Order assets in the asset collection
+
+We know that ordering assets can be important for you. Indeed, it can help you to **organize assets for your different channels**. Until the 2.2, it was impossible to do that in the PIM : assets were organized only according to their code.
+From the 2.3 version, we added a simple way to do that, directly from the product form.
+
+OK, let's see !
+
+![drag_drop](../img/drag_drop.jpg)
```
Comment (KmilleF) [inline content/md/products-super-power/work-on-a-product.md]: did you drop the isea of animated GIF?

Comment (KmilleF) [reply content/md/products-super-power/work-on-a-product.md]: *idea

File: content/md/products-super-power/work-on-a-product.md [hunk 7]
```diff
@@ -95,6 +95,76 @@ From the product form:
...
+
+You need to import assets in a certain order in Akeneo PIM ? No problem : now you can ! Just order your assets in your import file, separating asset's names with a coma, then retrieve this order in the PIM.
+
+The order defined in the PIM will also be the same in the API.
+
+::: tips
+And if you want to have a look at history, a new version is created in `History` tab each time the user changes the asset order.
+:::
+
+### Better assets display
```
Comment (KmilleF) [inline content/md/products-super-power/work-on-a-product.md]: is is still in the product form or in the PAM?

Comment (KmilleF) [reply content/md/products-super-power/work-on-a-product.md]: got my answer, commented too fast ;)

File: content/md/products-super-power/work-on-a-product.md (hunk 4 above)
Comment (morganen) [reply content/md/products-super-power/work-on-a-product.md]: I change "retrieve" by "find"

Comment (morganen) [reply content/md/products-super-power/work-on-a-product.md]: I used "retrieve" 3 times so I change the word 3 times ;)

**Pull request merged**